go/vt/vttablet/grpctmclient/client.go

```diff
@@ -630,6 +630,7 @@ func (client *Client) PopulateReparentJournal(ctx context.Context, tablet *topod
 		ActionName:          actionName,
 		MasterAlias:         masterAlias,
 		ReplicationPosition: pos,
+		DbName:              topoproto.TabletDbName(tablet),
```

Comment:
I have a feeling reparent_journal should not have a db name.

Reply:
I see what you mean. Right now the keyspace-shard combination uniquely identifies both the vttablet and the mysql instance. The reparent happens at the mysql instance level, which means that withe multiple schemas on the same mysql, all keyspace-shards residing on that instance are affected. It seems to me that we need to introduce a way of uniquely identifying the mysql instance that sits at a higher level than keyspace-shard.

Reply:
I rebased and dropped the reparent_journal changes.